### ISSUE TITLE
SCAM!! DONT FCKING TRY

### ISSUE THREAD

**Issue by hohoanganhdung** [JsNodeDev/FrontrunDeployer#1]
this is scam everyone. you will lose your money. he made a code that automatically transferred money from your newly created contact to his wallet address!!

**Comment by CryptoMasterDegen** [comment]
yes ser he sure did

**Comment by CryptoMasterDegen** [comment]
refund my 0.30 dev 0x0D04e9432C6AA1729a504b9Fb05402d4336a4f24

**Comment by therothy** [comment]
What a scumbag

**Comment by CryptoMasterDegen** [comment]
Yeah he made a ton of BNB off this scam.  should have known too good to be true..   do the right thing and refund my 0.30 or have bad karma for the rest of your life.
